packages/bsky/src/services/indexing/processor.ts

```diff
@@ -230,6 +233,17 @@ export class RecordProcessor<T, S> {
       notifs = this.params.notifsForInsert(op.inserted)
     }
     for (const chunk of chunkArray(notifs, 500)) {
+      if (this.notifServer) {
```

Comment:
seems nicer to do this out in the indexing service instead of within the processor which I view as a "raw record -> DB indexes" translator 🤔 

any reason to prefer it here?

Reply:
I put it here in order mostly to make use of the existing code paths for notification handling, make it simpler to answer "where are notifications sent?", ensure the same logic was being applied no matter the destination of the notifications.  Previously notifications that may be generated from a deletion weren't being handled, and the added `notifsForInsert()` calls down in `insertRecord()`/`updateRecord()` felt a bit leaky/special-cased to me.

I do see what you're saying, though.  I may just have a slightly different perspective on it—that the destination of the notifications (in the db or off to some push service) isn't all that significant.

My guess is that we could get the best of both worlds with a little refactor!  But now might not be the time for it.  Happy to switch it back to the way it was if you think that's the move 👍

Reply:
nah that's cool I dig the justification 👍 

if we end up maturing this system, that might be a good time to move it around a bit